ghost/i18n/locales/hi/portal.json

```diff
@@ -0,0 +1,159 @@
+    "{{memberEmail}} will no longer receive this newsletter.": "{{memberEmail}} को अब यह न्यूज़लेटर प्राप्त नहीं होगा।",
+    "{{trialDays}} days free": "{{trialDays}} दिन मुफ्त",
+    "A login link has been sent to your inbox. If it doesn't arrive in 3 minutes, be sure to check your spam folder.": "एक लॉगिन लिंक आपके इनबॉक्स में भेजा गया है। यदि यह 3 मिनट में नहीं आता है, तो कृपया अपना स्पैम फ़ोल्डर जांचें।",
+    "Account": "खाता",
+    "Account settings": "खाता सेटिंग्स",
+    "After a free trial ends, you will be charged the regular price for the tier you've chosen. You can always cancel before then.": "मुफ्त परीक्षण समाप्त होने के बाद, आपको आपके द्वारा चुने गए स्तर के लिए नियमित कीमत पर शुल्क लिया जाएगा। आप तब तक किसी भी समय रद्द कर सकते हैं।",
+    "Already a member?": "पहले से ही सदस्य हैं?",
+    "An unexpected error occured. Please try again or <a>contact support</a> if the error persists.": "एक अप्रत्याशित त्रुटि हुई। कृपया फिर से प्रयास करें या यदि त्रुटि बनी रहती है तो <a>समर्थन से संपर्क करें</a>।",
+    "Back": "वापस",
+    "Back to Log in": "लॉगिन पर वापस जाएं",
+    "Billing info": "बिलिंग जानकारी",
+    "Black Friday": "ब्लैक फ्राइडे",
+    "Cancel anytime.": "कभी भी रद्द करें।",
+    "Cancel subscription": "सदस्यता रद्द करें",
+    "Cancellation reason": "रद्द करने का कारण",
+    "Change": "परिवर्तन",
+    "Check spam & promotions folders": "स्पैम और प्रचार फ़ोल्डरों की जांच करें",
+    "Check with your mail provider": "अपने मेल प्रदाता से जांचें",
+    "Choose": "चुनें",
+    "Choose a different plan": "एक अलग योजना चुनें",
+    "Choose your newsletters": "अपने न्यूज़लेटर चुनें",
+    "Click here to retry": "फिर से प्रयास करने के लिए यहां क्लिक करें",
+    "Close": "बंद करें",
+    "Comments": "टिप्पणियाँ",
+    "Complimentary": "नि:शुल्क",
+    "Confirm": "पुष्टि करें",
+    "Confirm cancellation": "रद्दीकरण की पुष्टि करें",
+    "Confirm subscription": "सदस्यता की पुष्टि करें",
+    "Contact support": "समर्थन से संपर्क करें",
+    "Continue": "जारी रखें",
+    "Continue subscription": "सदस्यता जारी रखें",
+    "Could not sign in. Login link expired.": "साइन इन नहीं कर सके। लॉगिन लिंक समाप्त हो गया।",
+    "Could not update email! Invalid link.": "ईमेल अपडेट नहीं कर सके! अमान्य लिंक।",
+    "Create a new contact": "एक नया संपर्क बनाएं",
+    "Current plan": "वर्तमान योजना",
+    "Delete account": "खाता हटाएं",
+    "Didn't mean to do this? Manage your preferences <button>here</button>.": "यह करने का मतलब नहीं था? अपनी प्राथमिकताएँ <button>यहाँ</button> प्रबंधित करें।",
+    "Don't have an account?": "क्या आपके पास खाता नहीं है?",
+    "Edit": "संपादित करें",
+    "Email": "ईमेल",
+    "Email newsletter": "ईमेल न्यूज़लेटर",
+    "Email preferences": "ईमेल प्राथमिकताएँ",
+    "Emails": "ईमेल",
+    "Emails disabled": "ईमेल निष्क्रिय",
+    "Ends {{offerEndDate}}": "{{offerEndDate}} को समाप्त होता है",
+    "Error": "त्रुटि",
```

Comment:
The translations are correct on paper but I feel that most of the hindi speaking people never use words like त्रुटि, संपादित, रद्दीकरण की पुष्टि, सहेजें etc, but still I have seen most websites using these kind of words. Makes it confusing for me to use a website in hindi. Just a thought, What do you think about that?

Reply:
I agree, I was doing translation for my site and did a rush job on some sections. I've changed them with better alternatives, please check and confirm if these look better.

Reply:
@vershwal , I think this is the only remaining question on this PR currently.  Just nudging this in case you missed the reply - can you mark the review as approved if your concerns are addressed?